uvicorn/main.py

```diff
@@ -319,6 +319,13 @@ def print_version(ctx: click.Context, param: click.Parameter, value: bool) -> No
     help="Ciphers to use (see stdlib ssl module's)",
     show_default=True,
 )
+@click.option(
+    "--ssl-options",
+    type=list,
+    default=None,
+    help="Options to set on ssl context",
```

Comment:
Those are integers, right? Can we point out the options available here as well?

```suggestion
    help="Options to set on ssl context.",
```

Reply:
added the file and Enum class to look for